### DISCUSSION TITLE
Nanopool : JSON data works inline but not from URL

### DISCUSSION THREAD

**Discussion by corgan2222** [grafana/grafana-infinity-datasource#105] [Specific APIs]
Thanks for this awesome Plugin! This is what i was looking for a long time!
But i have a Problem to get JSON data from a URL. If i copy the data direct to inline, it works fine. But from URL i get no Data.

URL to test: https://api.nanopool.org/v1/eth/hashratechart/0x6b69d1d3b0acd9c06617f3205ea13c58a5527464
any Ideas?

Inline
![grafik](https://user-images.githubusercontent.com/12233951/125452594-36a95adf-c5d5-44ec-bef5-aaf709bb1465.png)

URL:
![grafik](https://user-images.githubusercontent.com/12233951/125452638-b7ca5294-b2c9-4678-b108-827cd3b42754.png)

**Comment by yesoreyeram** [comment]
here is the sample. 

![image](https://user-images.githubusercontent.com/153843/125455992-350b8417-418c-4a17-83dc-48e8ff26e410.png)

This is exactly what you have shown. Using Grafana8.0.3 and Infinity0.7.5

What i think is, you might have URL specified in the datasource config as well. I would suggest leave the URL in the datasource config to blank and test. otherwise if you have specified `https://api.nanopool.org/v1` as the URL in the config editor, then in the query just specify the remaining part of the URL `/eth/hashratechart/0x6b69d1d3b0acd9c06617f3205ea13c58a5527464`

**Comment by corgan2222** [comment]
aahhh, the url don't get overridden as I was expecting. Thanks a lot, now it works great!
